ffmpeg/encoder.c

```diff
@@ -376,6 +379,23 @@ int mux(AVPacket *pkt, AVRational tb, struct output_ctx *octx, AVStream *ost)
   if (AVMEDIA_TYPE_AUDIO == ost->codecpar->codec_type) {
       if (octx->drop_ts == AV_NOPTS_VALUE) octx->drop_ts = pkt->pts;
       if (pkt->pts && pkt->pts == octx->drop_ts) return 0;
+
+      if (pkt->dts != AV_NOPTS_VALUE && pkt->pts != AV_NOPTS_VALUE && pkt->dts > pkt->pts) {
```

Comment:
WDYT about adding a permalink to the relevant code from the ffmpeg CLI that this is taken from as a reference?

Reply:
> Could a more descriptive name for audio-dts.ts be duplicate-audio-dts.ts?
> Why did the file permissions change for the sign_* files?
> WDYT about adding a permalink to the relevant code from the ffmpeg CLI that this is taken from as a reference?

Changed audio-dts.ts to duplicate-audio-dts.ts.
There is no special reason, just remove the execute permissions for those files.
Added permalink to the relevant code from the ffmpeg CLI.